plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -177,6 +177,21 @@
                 "reloaded"
             ],
             "description": "Toolkit run state."
+        },
```

Comment:
can this be added to common?

Reply:
I can follow up with a separate PR after this